src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Blocks/suppliers.html.twig

```diff
@@ -0,0 +1,62 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+
+{% set suppliersForm = productForm.suppliers %}
+
+<div class="row">
+  <div class="col">
+    {{ form_row(suppliersForm.supplier_ids) }}
+  </div>
+  <div class="col">
+    {{ form_row(suppliersForm.default_supplier_id) }}
+  </div>
+</div>
+
+{% for supplierId in suppliersForm.supplier_ids.vars.value %}
+  {% set productSuppliersCollectionName = 'product_suppliers_by_supplier_' ~ supplierId %}
+  <table class="table">
+    <thead class="thead-default">
+      <tr>
+{#        <th>{{ 'Product name'|trans({}, 'Admin.Catalog.Feature') }}</th>#}
+        <th>{{ 'Supplier reference'|trans({}, 'Admin.Catalog.Feature') }}</th>
+        <th>{{ 'Cost price (tax excl.)'|trans({}, 'Admin.Catalog.Feature') }}</th>
```

Comment:
Is this wording new? I only found "Cost price" alone

Reply:
copied from old product page for now, there it is (screenshot)
![Screenshot from 2021-01-08 15-16-23](https://user-images.githubusercontent.com/31609858/104019632-90389800-51c4-11eb-8910-33f8080a203c.png)

Reply:
Thanks! No idea why I cannot find the string in Crowdin, but I checked the specs and there is indeed a string "Cost (tax excl.)" on the Product page and from what I see from your screenshot, it should indeed be located in Admin.Catalog.Feature.

Reply:
I believe this new wording was introduced recently by @marionf Because the initial one didn't precise if the price was with or without taxes